examples/restaurant_visits/run_anonymize_values.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2022 OpenMined.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Demo of running PipelineDP locally, without any external data processing framework
```

Comment:
Can we describe the purpose here, e.g. "This demo applies differential privacy to data that has already been pre-aggregated. This relies on assumptions that the partitions keys are public, and that the sensitivities of the pre-aggregated values are known".

(We can keep mention of the local execution, but IMO this is more of a side thing not really important for this demo)